scripts/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+echo "Build the RWD Docker container."
+
+BUILD_COMMAND="docker build -t docker-rwd ."
+
+echo "$BUILD_COMMAND"
```

Comment:
Why `echo` and `eval` here?

I think just executing the command and using `set -x` at the top would produce similar behavior.